.github/workflows/allennlp.yml

```diff
@@ -0,0 +1,39 @@
+name: allennlp
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - 'allennlp/**'
+      - '.github/workflows/allennlp.yml'
+
+jobs:
+  examples:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Seems like there's a way to create a matrix over `runs-on` in which case we could iterate over prebuilt optuna official docker images https://github.community/t/create-matrix-with-multiple-os-and-env-for-each-one/16895/2. But this also adds another level of complexity so it's just a comment for the record.